flocker/node/agents/ebs.py

```diff
@@ -20,12 +20,15 @@
 from boto.utils import get_instance_metadata
 from boto.exception import EC2ResponseError
 from twisted.python.filepath import FilePath
-from eliot import Field, MessageType
 
 from .blockdevice import (
     IBlockDeviceAPI, BlockDeviceVolume, UnknownVolume, AlreadyAttachedVolume,
     UnattachedVolume, get_blockdevice_volume
 )
+from ._logging import (
+    AWS_ACTION, BOTO_EC2RESPONSE_ERROR, NO_AVAILABLE_DEVICE,
+    NO_NEW_DEVICE_IN_OS
+)
```

Comment:
What's your plan for the new `_logging` module?
Will we move all our Eliot definitions here? Including the ones already defined in `blockdevice` and `cinder`?
Which types, messages and actions can be shared between the implementations?
And `_logging` implies that its contents are private. Are there any which should be made public in `flocker.node.agents.__init__` ?